ship-it-inc/ship-it-fend
# #168234799 implement user dashboard

#### What does this PR do?
It implements the user dashboard
#### Description of Task to be completed?
- [x] add user dashboard
- [x] implement display user balance
- [x] implement display user order counts
#### How should this be manually tested?
- Fetch the branch `ft-implement-user-dashboard-168234799`
- Check out into the branch `ft-implement-user-dashboard-168234799`
- run npm run dev
- You should be to login with email and you will be directed to the user dashboard.
#### Any background context you want to provide? 
N/A
#### What are the relevant pivotal tracker stories?
[#168234799](https://www.pivotaltracker.com/story/show/168234799)
#### Screenshots (if appropriate)
N/A
